pkgs/racket-doc/xml/xml.scrbl

```diff
@@ -14,6 +16,10 @@
 @interaction-eval[#:eval xml-eval (require racket/list)]
 @interaction-eval[#:eval plist-eval (require xml/plist)]
 
+@(define-syntax-parse-rule (defthing/literal-value name:id contract #:from mod:id xs ...)
```

Comment:
I think you can avoid the redundant `#:from mod:id` by using the third result of `(identifier-binding #'name #f)`. (In the general case you might have to do something more to handle renaming, too &hellip;)